### PR TITLE
Remove "Back to Login" on Manage Account page

### LINKED ISSUE #3194: [Bug] Remove "Back to Login" on Manage Account page
**Description-**
The Back to Login link/button was added to the affiliation component but it may not make sense to have it on the manage account page since the assumption is they are already logged in-

**Type of Bug-**

- [ ] Accessibility
- [ ] Functional
- [x] Usability
- [ ] API
- [x] Regression

**Steps to reproduce-**

1. Step
2. Step
3. Step

**Role Affected-**

- [x] All
- [ ] Federal Admin
- [ ] State Admin
- [ ] State Staff
- [ ] State Contractor
- [ ] System Admin

**Environment-**

- [x] All
- [ ] PR #\_\_\_\_
- [ ] Staging
- [ ] Production

**Browser-**

- [x] Firefox
- [x] Chrome
- [x] IE

**Expected results-**

**Actual results-**

**Related user story/bug-** (if applicable)

**Additional information-** (optional)

**Logs or error output/Screenshot/Video-** (at least 1 of these)

---

Add additional labels (design, dev, compliance, BUG, etc) and size before submitting.

If the issue is needed to complete prioritized work for the CURRENT SPRINT, add it to the "This Sprint" pipeline. Otherwise, all other issues will be automatically added to the _unprioritized pipeline_ for prioritization in backlog refinement or sprint planning with Product.

### PR DESCRIPTION
**Pull request by knollfear**
resolves #3194 

**Description-**

Changes the text on the secondary button when the user is on the manage account screen

**This pull request changes...**

- Changes the text on the manage accoutn screen to match the text in the header
- DOES NOT change the text a user who has never requested any affiliations sees when they log in.

**This pull request also touches…**

- possible side effects of this change:

There are 2 separate user experiences here, one for someone who has no affiliations ever, this should be unchanged, and another for a user that has at least one current affiliation and is trying to add more.  If a user already has affiliations they should not see "Back to login" on the manage account screen, they should see words that match the header in the top left of the header.

**This pull request was tested in the follow ways…**

- manually testing of the relevant use cases
- updated a frontend test to ensure text was being passed forward to the secondary button

**Steps to manually verify this change...**

1. Log in as sysadmin and verify nothing has changed, including the function of the logout link
2. log in as a state staff user and click on "Manage Account" in the hamburger menu
3. verify that the "cancel" link text matches the header text
4. click the cancel text and you should be sent to the dashboard immediately with no loading or intermediate screens

### This pull request is ready to review when...

- [ ] Automated tests are updated (and all tests are passing)
- [ ] The change has been documented
- [ ] Associated OpenAPI documentation has been updated
- [ ] Changelog is updated as appropriate
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This pull request can be merged when…

- [ ] Code has been reviewed by someone other than the original author
- [ ] QA has verified the accessibility and functionality related to the change
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Review by thetif — CHANGES_REQUESTED** [review]

**Review by thetif — APPROVED** [review]
looks good!

File: web/src/containers/admin/ManageAccount.js [hunk 1]
```diff
@@ -32,6 +37,14 @@ const ManageAccount = ({
   if(error) {
     setShowConfirmation(false);
   }
+
+  const secondaryButtonText = isAdmin
+    ? 'Admin Dashboard'
+    : `< ${
```
Comment (thetif) [inline web/src/containers/admin/ManageAccount.js:43]: can you remove the <? it's not necessary here and it's inconsistent

**Review by thetif — APPROVED** [review]
Looks good. Testing in the following ways:

|Given|When|Then|Result|
|---|---|---|---|
|logged in as em@il.com|click on manage account|see "AK APD Home" in the cancel position and in the header|PASS|
|logged in as em@il.com and in manage account|click "AK APD Home" cancel|return to the dashboard|PASS|
|logged in as em@il.com and in manage account|click "AK APD Home" in header|return to the dashboard|PASS|
|logged in as fedadmin|click on manage account|see "Admin Dashboard" in the cancel position and in the header|PASS|
|logged in as fedadmin and in manage account|click "Admin Dashboard" cancel|return to the dashboard|PASS|
|logged in as fedadmin and in manage account|click "Admin Dashboard in header|return to the dashboard|PASS|

File: web/src/containers/admin/ManageAccount.js (hunk 1 above)
Comment (knollfear) [reply web/src/containers/admin/ManageAccount.js]: It has been removed

**Review by beparticular — APPROVED** [review]

**Review by jeromeleecms — APPROVED** [review]
Looks good.

**Pull request merged**